dashboard/controller/users.js

```diff
@@ -294,3 +297,10 @@ function getClassrooms(req, callback){
 		}
 	});
 }
+
+/**
+ * private function - Regex validation
+ */
+function regex(type){
```

Comment:
2 comments
1. move it to helper/utils files
2. use switch statement to make it more generic.